families/supplychain/restapi/openapi.yaml

```diff
@@ -0,0 +1,377 @@
+swagger: '2.0'
+
+info:
+  version: "0.8.0"
+  title: Sawtooth Supplychain REST API
+  description: |
+    _This HTTP pragmatic REST API is built on top of Sawtooth Lake's existing
+    ZMQ/Protobuf infrastructure, simplifying SupplyChain clients interaction
+    with the blockchain by exposing endpoints that use common HTTP(S)/JSON
+    standards. The SupplyChain REST API provides secure means for clients to
+    access SupplyChain records held in global state in a easy manner. The
+    rest api provides retrieval using natural keys of the the records.
+    provides indexing and query capabilities for Agents, Applications, and
+    Records stored with in._
+
+paths:
+  /agents/:
+    get:
+      summary: Fetches the data for the current set of agents in state
+      description: >
+        Fetches a paginated list of leaves for the current state, or relative
```

Comment:
It doesn't seem like the domain specific representation should know anything about addresses or the fact that things are stored in a tree underneath.